pandas/tests/test_resample.py

```diff
@@ -3030,7 +3030,7 @@ def test_nearest(self):
         # GH 17496
         # Resample nearest
         index = pd.date_range('1/1/2000', periods=3, freq='T')
-        result = pd.Series(range(3), index=index).resample('20s').nearest()
+        result = pd.Series(np.arange(3), index=index).resample('20s').nearest()
```

Comment:
so I would actually leave *some* tests with pure range in the constructor, e.g. is there a reason to change this one?

Reply:
The expected is `pd.Series(np.array([0, ...]), index=...)`, and on Windows the `np.array` defaults to `int32` dtype, so the test fails since result is now `int64` dtype.

Either need `range` -> `np.arange` or `np.array` -> `list` (or specify the dtype in `np.array`) for the test to pass.  Changed it to keep `range` here.